diff --git a/node_modules/@photon-ai/imessage-kit/dist/index.cjs b/node_modules/@photon-ai/imessage-kit/dist/index.cjs
index 8e4c733..8a140ba 100644
--- a/node_modules/@photon-ai/imessage-kit/dist/index.cjs
+++ b/node_modules/@photon-ai/imessage-kit/dist/index.cjs
@@ -407,7 +407,7 @@ var MESSAGE_FIELDS = [
   "message.date_played",
   "message.date_edited",
   "message.date_retracted",
-  "message.date_recovered",
+  "NULL AS date_recovered",
   "message.is_empty",
   "message.message_summary_info",
   "message.reply_to_guid",
@@ -416,7 +416,7 @@ var MESSAGE_FIELDS = [
   "message.expressive_send_style_id",
   "message.balloon_bundle_id",
   "message.destination_caller_id",
-  "message.ck_chat_id as ck_chat_id",
+  "NULL AS ck_chat_id",
   "message.was_detonated",
   "message.expire_state",
   "message.share_status",
diff --git a/node_modules/@photon-ai/imessage-kit/dist/index.js b/node_modules/@photon-ai/imessage-kit/dist/index.js
index 184d04c..099593b 100644
--- a/node_modules/@photon-ai/imessage-kit/dist/index.js
+++ b/node_modules/@photon-ai/imessage-kit/dist/index.js
@@ -404,7 +404,7 @@ var MESSAGE_FIELDS = [
   "message.date_played",
   "message.date_edited",
   "message.date_retracted",
-  "message.date_recovered",
+  "NULL AS date_recovered",
   "message.is_empty",
   "message.message_summary_info",
   "message.reply_to_guid",
@@ -413,7 +413,7 @@ var MESSAGE_FIELDS = [
   "message.expressive_send_style_id",
   "message.balloon_bundle_id",
   "message.destination_caller_id",
-  "message.ck_chat_id as ck_chat_id",
+  "NULL AS ck_chat_id",
   "message.was_detonated",
   "message.expire_state",
   "message.share_status",
